GovUkDesignSystem/GovUkDesignSystemComponents/CookieBanner.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GovUkDesignSystem.GovUkDesignSystemComponents.CookieBannerViewModel
+
+@{
+    var message = Model.CookieType switch {
+        CookieType.Analytics => "We'd also like to use analytics cookies so we can understand how you use the service and make improvements.",
+        CookieType.Additional => "We'd like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.",
+        _ => throw new ArgumentOutOfRangeException()};
+    var decision = Model.BannerState switch {
+        BannerState.ShowAccepted => "accepted",
+        BannerState.ShowRejected => "rejected",
+        _ => null 
+        };
+}
+
+@if (Model.BannerState is not BannerState.Hide)
+{
+    <form method="POST" action="@Model.ButtonClickAction">
+        @if (Model.AntiForgeryToken)
```

Comment:
We should probably get rid of this `if` ASP.NET projects should generally be using anti forgery tokens

Reply:
I'll leave the comment and the merging to you then :)